### DISCUSSION TITLE
p/8ecc929074874291b7b1e7090d63349c/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#24] [Announcements]
# p/8ecc929074874291b7b1e7090d63349c/

这周决定五一回广州了，或许可以和朋友一起约饭，感觉有快两年不见了呢。看起来五一出行的人真的很多，高铁票真的很不好买，所以吾辈买了机票，虽然时间上也不是挺满意就是了。  开发由于信用卡上周末被封禁，所以这周有点没有动力继续尝试 gpt 相关工具开发，而且看起来短期内都会处于莽荒时期，它的优先级将不会是最高的了。 学习好吧，这周没那么疯狂，因为实在太卷了，上周吾辈卷到了 6k+ 就已经感觉很多了，但这

https://blog.rxliuli.com/p/8ecc929074874291b7b1e7090d63349c/

**Comment by miaopass-future** [comment]
时间安排的很好啊，有空出去玩，还能那么卷

**Reply by rxliuli** [reply]
出去玩是每周计划中的事情，至于卷，最近两周真的到了倦怠期，并不真的在卷
